constants.py

```diff
@@ -1,5 +1,5 @@
-JSON_PATH = './fyi_links.json'
-NETFLIX_SALARY_LINK = 'salary_link'
-RECENT_SALARY_XPATH = 'salary_xpath'
-RECENT_YOE_XPATH = 'yoe_xpath'
-RECENT_LEVEL_NAME_XPATH = 'level_xpath'
```

Comment:
LOL I always thought Python standard was single-quotes

Reply:
I think it depends on the context

Reply:
it is single :^)

Reply:
According to PEP, no preference. Just pick one and stick to it I guess? 

https://www.python.org/dev/peps/pep-0008/#string-quotes